### ISSUE TITLE
Tests fail (with recent Date::Holiday::CZ versions?)

### ISSUE THREAD

**Issue by eserte** [smithfarm/dochazka-rest#76]
The t/unit/holiday.t test started to fail on my smoker systems. Statistical analysis suggests that the problem started with Date::Holidays::CZ 0.10:

```
****************************************************************
Regression 'mod:Date::Holidays::CZ'
****************************************************************
Name                   Theta          StdErr     T-stat
[0='const']           1.0000          0.0000    58297932825350064.00
[1='eq_0.10']        -1.0000          0.0000    -17780702818424218.00
[2='eq_0.11']        -1.0000          0.0000    -39786082966060968.00

R^2= 1.000, N= 77, K= 3
****************************************************************
```

**Comment by smithfarm** [comment]
Thanks for the heads-up! Will get on this soon.

**Comment by smithfarm** [comment]
Fixed in `Date::Holidays::CZ` 0.12
